### DISCUSSION TITLE
图标的目录无法编译“\citep{}"

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#116] [Q&A]
你好，
图标的目录，无法编译`\citep{}`，图标的标题是正确的，但是图表的目录直接使用了文献的标签。
![Screenshot 2022-04-12 215303](https://user-images.githubusercontent.com/45897471/162978075-948be738-faa6-4364-aeff-2ee34fcf4c49.jpg)

![Screenshot 2022-04-12 203459](https://user-images.githubusercontent.com/45897471/162977912-810cbec3-48f4-4e9c-8608-3caf63dcbbf0.jpg)
这个问题可以修复吗？

**Comment by sikouhjw** [comment]
`\caption[正文标题]{正文标题\citep{xxx}}`

**Reply by earthmoon** [reply]
有没有办法把引用也代入目录中？

**Reply by sikouhjw** [reply]
> 有没有办法把引用也代入目录中？

错误的需求

**Reply by earthmoon** [reply]
> > 有没有办法把引用也代入目录中？
> 
> 错误的需求

好的吧。

**Reply by earthmoon** [reply]
[感觉是我们的模板限制了，因为我测试的时候是可行的。测试文件：](https://bthc-my.sharepoint.com/:f:/g/personal/mr_iem_im/EjLYEtGLaDBAp_yPdUQgJBkBHsed_059uZzUN7Y83e6gvg?e=3WCrVd) 

> https://bthc-my.sharepoint.com/:f:/g/personal/mr_iem_im/EjLYEtGLaDBAp_yPdUQgJBkBHsed_059uZzUN7Y83e6gvg?e=3WCrVd

**Reply by sikouhjw** [reply]
> 感觉是我们的模板限制了，因为我测试的时候是可行的。

在图表目录中，不应该有引用。在顺序编码制下，第一个引用的是在图表目录，你说这合理吗？

如果你非要纠结，建议不要写在 `\caption` 里，而是在正文描述的时候引用。

**Reply by earthmoon** [reply]
是的。我只是好奇，在空白模板可以实现这个功能，学校的模板不行。所以觉得是模板限制了这个功能。

**Reply by tanukihee** [reply]
我不知道我哪里有意识地限制了这一点，如果你能找到，欢迎 pr

**Reply by earthmoon** [reply]
我这水平用都用不来，就看他们那些大神了。
